Patches/Android Tiers/ThingDef_Misc/Weapons_Guns.xml

```diff
@@ -0,0 +1,359 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success></success>
+		<operations>
+			<li Class="PatchOperationFindMod">
+			<mods><li>[1.0] Android tiers</li></mods>
+			<match Class="PatchOperationSequence">
+			<operations>
+			<!-- ==========  40mm Cannon  =========== -->
+			
+			<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+			<defName>Mech40MMCannon</defName>
+				<statBases>
+					<SightsEfficiency>2.18</SightsEfficiency>
+					<ShotSpread>0.15</ShotSpread>
+					<SwayFactor>1.72</SwayFactor>
+					<Bulk>35</Bulk>
+					<Mass>42</Mass>
+					<RangedWeapon_Cooldown>0.37</RangedWeapon_Cooldown>
+				</statBases>
+				
+				<Properties>
+					<recoilAmount>1.72</recoilAmount>
+					<verbClass>CombatExtended.Verb_ShootCE</verbClass>
+					<hasStandardCommand>true</hasStandardCommand>
+					<defaultProjectile>Bullet_40x53mmGrenade_HE</defaultProjectile>
+					<warmupTime>1.45</warmupTime>
+					<burstShotCount>6</burstShotCount>
+					<ticksBetweenBurstShots>10</ticksBetweenBurstShots>
+					<range>86</range>
+					<soundCast>40MMCannon</soundCast>
+					<soundCastTail>GunTail_Heavy</soundCastTail>
+					<muzzleFlashScale>18</muzzleFlashScale>
+				</Properties>
```

Comment:
Explosive weapons should have the following element set, to let them target the ground:
```
<targetParams>
    <canTargetLocations>true</canTargetLocations>
</targetParams>
```